pkg/modules/authorization.go

```diff
@@ -114,6 +125,7 @@ var (
 	pathType    = networking.PathTypePrefix
 	duration    = 2160 * time.Hour // 90d
 	renewBefore = 360 * time.Hour  // 15d
+	replicas    = 5
```

Comment:
I think we should parameterize this, just so that if replicas are changed, then we can adjust accordingly.

Reply:
Made the value dynamic by passing the replica count value from the yaml file